features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,7 @@
+Given("That I am on the landing page") do
+    visit root_path  
+end
+
```

Comment:
Perhaps we should be cleaner here and have consistency in the .feature definitions - for example just use "I visit the landing page" and then we only need one step.

Reply:
fixed